scripts/metabiome.sh

```diff
@@ -48,6 +60,9 @@ Usage: metabiome [Commands|Options]
     -h, --help    Show this help.
     -v, --version Show Metabiome's version.
 
+If you are having problems running Metabiome, please do not hesitate to contact us through
+github (https://github.com/Nesper94/Metabiome)
```

Comment:
Same line length suggestion:
```suggestion
If you are having problems running Metabiome, please do not hesitate to
contact us through GitHub (https://github.com/Nesper94/Metabiome)
```